openpbs.spec

```diff
@@ -145,8 +145,11 @@ utilization & efficiency, and simplify administration for
 HPC clusters, clouds and supercomputers.
 
 %package %{pbs_server}
-Summary: PBS Professional for a server host
+Summary: OpenPBS for a server host
 Group: System Environment/Base
+Conflicts: openpbs-execution
+Conflicts: openpbs-client
+Conflicts: pbspro-server
 Conflicts: pbspro-execution
 Conflicts: pbspro-client
 Conflicts: pbspro-client-ohpc
```

Comment:
We should add additional conflicts here for openpbs-clients-ohpc (clients, server, etc.) since those will start being produced once the OpenHPC spec file gets updated.  

Same comment applies to other "Conflicts" sections in file.